packages/algorithm/tests/rollup.config.js

```diff
@@ -0,0 +1,10 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+
+export default {
+  input: './lib/index.spec.js',
+  output: {
+    file: './lib/bundle.test.js'
+  },
+  plugins: [commonjs(), nodeResolve()]
```

Comment:
Without `nodeResolve` plugin, the test files (*.spec.ts) cannot import from `chai` or `@lumino/algorithm`.

Without `commonjs` plugin, the `chai` import doesn't work. You get the following error:

```
Error: 'default' is not exported by ../../../node_modules/chai/index.js, imported by ../../../node_modules/chai/index.mjs
```

Note that with these two plugins, you will get a warning about circular dependencies in Chai, but it's just a warning, not an error, and the tests will build, run, and pass.

Does the order of these plugins matter?

Reply:
I wouldn't worry the plugin order for tests